cicd/pipelines/remote-pipelines-tasks-resolvers.adoc

```diff
@@ -23,6 +23,11 @@ Git resolver:: Retrieves a task or pipeline binding from a Git repository. You m
 == Specifying a remote pipeline or task from a Tekton catalog
 You can specify a remote pipeline or task that is defined in a public Tekton catalog, either link:https://artifacthub.io/[{artifact-hub}] or link:https://hub.tekton.dev/[{tekton-hub}], by using the hub resolver.
 
+[IMPORTANT]
+====
+The {artifact-hub} product is not supported with {pipelines-title}. Only the configuration of {artifact-hub} is supported.
+====
```

Comment:
I don't think Artifact Hub is a product to start with. Do you would want to say something like "The tasks and pipelines available on Artifact Hub are not supported..."? Or do you want to say "Deploying a custom instance of Artifact Hub is not supported..."?

Reply:
@mramendi we want to support Artifact as hub resolver in task or pipelines.
But we will not support, if there is an issue reported by customer in Artifact Hub product itself.

cc @vdemeester

Reply:
@jkandasa @vdemeester I am not sure how Artifact Hub can be described as a product

Reply:
@mramendi Artifact Hub maintained out side of tekton community.
and It is a web-based application. So I called it as a product.
* https://artifacthub.io/
* https://artifacthub.io/docs/

May be the `project` is the right word?
You can rename it as required. Thanks @mramendi